content/_index.md

```diff
@@ -10,3 +10,22 @@ Welcome to the website of the NWERC 2022!
 The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The team that solves the most problems at the end of the contest qualifies for the ICPC World Finals.
 
 The 2022 edition of the Northwestern European Regional Contest (NWERC) will be held on the 25th of November until the 27th of November 2022. The contest will be held on the campus of the Delft University of Technology.
+
+{{% row %}}
+{{% column %}}
+
+## Contest material
+
+- [Livestream (contest)](https://youtu.be/aDsW8J8P7go)
```

Comment:
Wouldn't it be nicer to read this the other way around? "Contest livestream" sounds more sensible and makes for an easier distinction between all the links. Maybe even make a header "Livestreams" instead of adding that to the link?

Reply:
Reading it the other way around sounds nice, done :slightly_smiling_face: 

I want to leave the headers like this, because:

> We can use these sections to later add the scoreboard, problem set, solutions, etc.

Reply:
I thought maybe a subheading for the livestreams, not changing the ones that are there currently